### ISSUE TITLE
Repurpose vertical bars in gutter as a diff since opening the document.

### ISSUE THREAD

**Issue by luckydonald** [hedgedoc/hedgedoc#5888]
### Which part of the project should be enhanced?

editor

### Is your enhancement request related to a problem? Please describe.

When I'm working with someone else, and want to know what the changes they made in a glance.
I assumed that's the markings on the side of the code lines, but that's not entirely true, see #2892, it's only the last change, they will never go away.

### Describe the solution you'd like

So I would like keep the visuals, but give them a new purpose:
Changes by others since I opened up the page.
Basically lines which were changed since the page load would be marked in a color fitting to the user (like the current implementation)
However, they would vanish once I reload the page, hence being completely client side.

### Describe alternatives you've considered

1. Keep the current implementation (which #2892 would change)
2. Having a fully fledged out interface for filtering changes by a person and time
3. Having a specific "mark changes mode" like word has
